app/src/androidTest/java/com/processout/processoutexample/ExampleInstrumentedTest.java

```diff
@@ -19,8 +20,7 @@
     @Test
     public void useAppContext() throws Exception {
         // Context of the app under test.
-        Context appContext = InstrumentationRegistry.getTargetContext();
-
+        Context appContext = appContext = InstrumentationRegistry.getInstrumentation().getTargetContext();
```

Comment:
I'm not a Java developer so I apologise if I've misunderstood this syntax, but wouldn't these be equivalent? 

```suggestion
        Context appContext = InstrumentationRegistry.getInstrumentation().getTargetContext();
```

Reply:
Yes, it would be the same. But it does not matter as it is just an example test. 
I could "fix" it in the next PR or even better remove this class as it is provided just as a template by Android Studio by default and does not any value.